portfolio/src/main/webapp/experience.html

```diff
@@ -3,16 +3,21 @@
   <head>
     <meta charset="UTF-8">
     <title>Experiences</title>
-    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css/nav.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="nav">
-      <ul>
+    <nav>
+      <div class="logo">
+          <h1>Winnie Wan</h1>
+      </div>
+      <ul class="nav-li">
         <li><a href="index.html">Home</a></li>
-        <li><a href="experience.html">Experiences</a></li>
+        <li class="active"><a href="experience.html">Experiences</a></li>
```

Comment:
do you want to add this styling to your other pages as well?

Reply:
Yes. I just merged it with master page to retrieve other pages.